www/js/lib/uiUtil.js

```diff
@@ -42,6 +42,36 @@ define([], function() {
         });
         jQueryNode.attr(nodeAttribute, url);
     }
+
+    /**
+     * Replace the given CSS link (from the DOM) with an inline CSS of the given content
+     * 
+     * For some reason, Firefox OS does not accept the syntax <link rel="stylesheet" href="data:text/css,...">
```

Comment:
@mossroy I think the reason is CSP (Content Security Policy). See  https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Firefox_OS_apps/Building_apps_for_Firefox_OS/CSP . Should we change this comment to:
"Due to CSP, Firefox OS...."?

Reply:
You're right : adding a link to this page gives more explanation on the reason.
It's not because the browser engine is old, but because of some CSP restrictions that have been put intentionally on it.
And some other platforms (like browser extensions) can apply some CSP restrictions : https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Content_Security_Policy

Reply:
All right, no problem